test/unit/spec/IDPDiscovery_spec.js

```diff
@@ -1036,5 +1046,47 @@ function (_, $, Q, OktaAuth, LoginUtil, Okta, Util, AuthContainer, IDPDiscoveryF
         });
       });
     });
+
+    Expect.describe('Registration Flow', function () {
+      itp('does not show the registration button if features.registration is not set', function () {
+        return setup().then(function (test) {
+          expect(test.form.registrationContainer().length).toBe(0);
+        });
+      });
+      itp('does not show the registration button if features.registration is false', function () {
+        var registration =  {
+        };
+        return setupRegistrationButton(null, registration).then(function (test) {
```

Comment:
description says `features.registration` is false, but here we are passing `null`. Should we change it to `false` or change the test description to match the test?